bee-common/bee-packable/tests/bounded.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod common;
+
+use bee_packable::{
+    Bounded, BoundedU16, BoundedU32, BoundedU64, BoundedU8, InvalidBoundedU16, InvalidBoundedU32, InvalidBoundedU64,
+    InvalidBoundedU8, Packable, UnpackError,
+};
+
+use core::convert::TryFrom;
+
+macro_rules! impl_bounds_test_for_bounded {
+    ($name:ident, $wrapper:ty, $error:ident) => {
+        #[test]
+        fn $name() {
+            let valid = <$wrapper>::MIN;
+            assert!(<$wrapper>::try_from(valid).is_ok());
```

Comment:
Can you add a stupid test that checks that the inner value is actually what it's supposed to be please ?